src/server/api/schema.js

```diff
@@ -1298,6 +1299,52 @@ const rootMutations = {
         .where("id", id)
         .update({ is_deleted: true });
       return { id };
+    },
+    buyPhoneNumbers: async (
```

Comment:
@lperson has started what might be a good trend in creating these as mutations/ files rather than continue making the large schema.js file even bigger.  

Optional, but if you move it to one, I think that would be good.  If you do so, please move the whole function, so you can import and then just include `buyPhoneNumbers` as a unitary key-value in the list.

Reply:
nice, i definitely prefer this pattern